requirements.txt

```diff
@@ -1,5 +1,5 @@
 # NOTE: This file is parsed by .ci/generate_bear_requirements.py
-# Use >= for development versions so that source builds always work
+# Edit coala_dependency in coala-build.yaml
```

Comment:
this should be called `coalib_dependency`, and should reside in the local yaml, not in the global yaml.

Reply:
The intention of using a global `coala_dependency` is that at release time all repos should switch to using the latest, and not have a local override pinning them to an old version.

The best way to do that is to update the global `coala_dependency` to a pre-release , watch stuff break, fix stuff, and iterate until nothing is broken.

The problem is that the bears repo often needs to use a more recent coala pre-release , and that change would flow to all other repos.  I guess that isnt so bad, as it ensures all repos are in sync more often, and coala core hasnt broken something drastic.